tests/contrib/pymongo/test.py

```diff
@@ -1,7 +1,7 @@
 # stdlib
+# 3p
```

Comment:
nit: So this only applies for `time`? Since we're using our local wrapped `pymongo`?

Reply:
Also didn't know we did this 😅

Reply:
This is from linting this really old file. I didn't add this comment. We can remove it.